tests/test_sp_multiheadattention.py

```diff
@@ -0,0 +1,46 @@
+import torch
```

Comment:
let's move this `tests/nn/data_parallel/test_sp_m...py`.

Reply:
I moved this file in `tests/nn/data_parallel`